content/backend/graphql-js/4-adding-a-database.md

```diff
@@ -242,15 +242,15 @@ Unfortunately, if you do so you will be greeted by an error:
 }
 ```
 
-Remember how we said that your Prisma API is protected by the `secret` from `prisma.yml`? Well, this is precisely the reason why you're getting the error right now. The Playground is trying to load the GraphQL schema from the endpoint, but its request is not authenticated. Let's go ahead and change that.
+Remember how we said that your Prisma API is protected by the `secret` from `prisma.yml`? Well, this is precisely the reason why you're getting the error right now. The Playground is trying to load the GraphQL schema from the endpoint, but it's request is not authenticated. Let's go ahead and change that.
```

Comment:
I guess that `its` is actually the correct one, since it's referring to the Playground's request.